src/sass/modules/_m-growable.scss

```diff
@@ -0,0 +1,6 @@
+.va-growable {
```

Comment:
I don't know what our preference here is, but I'm not a huge fan of nesting and styling generic class names like this. It ends up causing specificity issues as your codebase grows. My preference is for something like `va-growable-row` or `va-growable-edit`.